src/services/attendees/attendee-schema.ts

```diff
@@ -0,0 +1,20 @@
+import mongoose from "mongoose";
+import { z } from "zod";
+
+
+// Zod schema for attendee
+const CreateAttendeeSchema = z.object({
+    name: z.string(),
```

Comment:
this looks good for now, but once we've finalized the registration schema, the attendee schema should look very similar because they both should share fields (ex. university affiliation, dietary restrictions, gender, etc).